packages/wallet-adapter-core/src/constants.ts

```diff
@@ -23,3 +23,8 @@ export enum NetworkName {
   Testnet = "testnet",
   Devnet = "devnet",
 }
+
+export const ChainIdToAnsContractAddressMap: Record<string, string> = {
+  "1": "mainnet", // mainnet
+  "2": "testnet", // testnet
+};
```

Comment:
it says "chain id to contract address" but the map points to a network name?

Reply:
yep - will update